### ISSUE TITLE
Issues building on 10.11.2

### ISSUE THREAD

**Issue by jfktrey** [x43x61x69/gbdk-osx#1]
When trying to build, I receive the following error:

```
gcc -o ../../bin/makebin makebin.o
ld: can't open output file for writing: ../../bin/makebin, errno=2 for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [../../bin/makebin] Error 1
make[1]: *** [sdcc-libs] Error 2
make: *** [sdcc-build] Error 2
```

Simply ran `make` inside the root directory.

**Comment by tomekc** [comment]
I ran into the same issue. The problem may lie in hardcoded paths in makefiles.
First I tried to use sdcc compiler installed by **brew**, and build library on my own. It turned out that GBDK is not compiling with recent versions od sdc (I have 3.4.x).

Instead of trying to adjust makefiles or sources, I did a research and found that project:

https://github.com/rotmoset/gbdk-n

Compiles nicely with sdcc from brew, and seems to be maintained.
